tests/regress/high_mem.py

```diff
@@ -23,8 +21,10 @@ def map_code_page(self, address, payload):
         self.uc.mem_map(address, 0x1000)
         self.uc.mem_write(address, payload)
 
+    @unittest.skipIf(sys.version_info < (3, 7), reason="requires python3.7 or higher")
     def test_virt_high_mapping(self):
-        """Mapping memory at high addresses should work when TLB mode
+        """
```

Comment:
Not sure why this one was changed. PEP8 (and originally PEP 257) states that docstrings should start in the same lines with the quotes.